### ISSUE TITLE
Document the build, release, test process

### ISSUE THREAD

**Issue by hobbit1983** [galasa-dev/projectmanagement#1112]

**Comment by jadecarino** [comment]
@hobbit1983 Hi Will, is this issue referring to the release process for a 0.xx.0 version of Galasa?

**Comment by hobbit1983** [comment]
Yes - we were going to ask you to do a release of Galasa so that you could then document it

**Comment by jadecarino** [comment]
@hobbit1983 @techcobweb Hi both, while going through the release I documented the new process here: https://github.com/galasa-dev/automation/tree/main/releasePipeline

Would you say this is sufficient to close off this item?

**Comment by jadecarino** [comment]
I have added some more detail to the release documentation to describe how to work around some of the issues I encountered in the v0.25.0 release, that we currently do not yet have a fix for. If we have fixed these problems before the v0.26.0 release, we can ignore these steps and do it the automated way. If not, these manual steps should help whoever releases next to complete the release.

**Comment by techcobweb** [comment]
I think it's looking like a good level of detail. Won't know if the instructions work until someone tries to use them though. Lets close and open defects if we later find anything ?

**Comment by jadecarino** [comment]
Thanks Mike. These are actually the instructions I used to release , minus the extra temporary steps I added after encountering some problems, but these weren't to do with the instructions just small problems in the automation. So they worked for me but yes would be good for someone else to give their opinion when they use them. Closing this for now, and I'll reopen if we find problems later as you mentioned :) Assigning to this sprint and story pointed as 1 to claim the time for the amendments I made.
